go/vt/discovery/healthcheck.go

```diff
@@ -187,6 +190,24 @@ func (e *TabletStats) DeepEqual(f *TabletStats) bool {
 			(e.LastError != nil && f.LastError != nil && e.LastError.Error() == f.LastError.Error()))
 }
 
+// GetTabletHostPort Formats a tablet host port address
+func (e TabletStats) GetTabletHostPort() string {
+	vtPort := e.Tablet.PortMap["vt"]
+	return netutil.JoinHostPort(e.Tablet.Hostname, vtPort)
+}
+
+// Formats a debug url to the tablet
+func (e TabletStats) getTabletDebugURL() string {
+	tmpl := template.New("")
+	tmpl, err := tmpl.Parse(*tabletURLTemplate)
+	if err != nil {
+		log.Fatalf("error parsing template: %v", err)
```

Comment:
This is not good because you would crash the vtgate at runtime.

Instead, you could make the template a global variable and set it during init().

Also, please use `Exitf` instead of `Fatalf`: https://godoc.org/github.com/golang/glog#Exitf

Reply:
Yeah thanks for the tip. It's been fixed I think. I'm clearly still green behind the ears.